R/dstrace.R

```diff
@@ -153,8 +101,8 @@ dstrace <- function(object_name, direction = "upstream", graph = FALSE,
 #'
 #' @param object_name Name of original object being traced
 #' @param tracelist Record of the trace, a tibble
-#' @param upstream Upstream? Logical
-#' @param downstream Downstream? Logical
+#' @param upstream Looking upstream? Logical
+#' @param downstream Looking downstream? Logical
```

Comment:
Took me a bit to figure out what "looking upstream" / "looking downstream" meant. Maybe "Return upstream precursors?" and "Return downstream dependencies?"